src/Resources/views/includes/_action.html.twig

```diff
@@ -154,10 +154,13 @@
 
 {% block modal_action %}
     {# @var action \araise\CrudBundle\Action\Action #}
+    {% set attr = {
+        class: 'whatwedo-crud-button--action group flex items-center px-4 py-2 ml-0',
+        type: 'button',
+    }|merge(action.option('attr')|default([])) %}
```

Comment:
if the button defines a custom css class like following:
```
Action::OPT_ATTR => [
         'data-test' => 'test',
         'class' => 'my-custom-class',
    ]
```
our default should be merged with the given one. See example here: https://github.com/whatwedo/CrudBundle/blob/1.0-dev/src/Resources/views/includes/layout/_block.html.twig#L6

Reply:
@tuxes3 is this what you had in mind?
https://github.com/araise-dev/CoreBundle/blob/270da5bf22861bd952daba64abc5d4683f01f43f/src/Resources/views/includes/_action.html.twig#L161